doubleml/double_ml_score_mixins.py

```diff
@@ -0,0 +1,218 @@
+                    b = np.minimum(coef_start + delta * max_bracket_length/2, coef_bounds[1])
+                    b_guess = (a, b)
+                    f_a = score(b_guess[0])
+                    f_b = score(b_guess[1])
+                    s_different = (np.sign(f_a) != np.sign(f_b))
+                    delta += 0.1
+                return s_different, b_guess
+
+            signs_different, bracket_guess = get_bracket_guess(self._coef_start_val, self._coef_bounds)
+
+            if signs_different:
+                root_res = root_scalar(score,
+                                       bracket=bracket_guess,
+                                       method='brentq')
+                theta_hat = root_res.root
+            else:
+                # try to find an alternative start value
+                def score_squared(theta):
+                    res = np.power(np.mean(self._compute_score(psi_elements, theta)), 2)
+                    return res
+                # def score_squared_deriv(theta, inds):
+                #     res = 2 * np.mean(self._compute_score(psi_elements, theta, inds)) * \
+                #           np.mean(self._compute_score_deriv(psi_elements, theta, inds))
+                #     return res
+                alt_coef_start, _, _ = fmin_l_bfgs_b(score_squared,
+                                                     self._coef_start_val,
+                                                     approx_grad=True,
+                                                     bounds=[self._coef_bounds])
+                signs_different, bracket_guess = get_bracket_guess(alt_coef_start, self._coef_bounds)
+
+                if signs_different:
+                    root_res = root_scalar(score,
+                                           bracket=bracket_guess,
+                                           method='brentq')
+                    theta_hat = root_res.root
+                else:
+                    score_val_sign = np.sign(score(alt_coef_start))
+                    if score_val_sign > 0:
+                        theta_hat, score_val, _ = fmin_l_bfgs_b(score,
+                                                                self._coef_start_val,
+                                                                approx_grad=True,
+                                                                bounds=[self._coef_bounds])
+                        warnings.warn('Could not find a root of the score function.\n '
+                                      f'Minimum score value found is {score_val} '
+                                      f'for parameter theta equal to {theta_hat}.\n '
+                                      'No theta found such that the score function evaluates to a negative value.')
+                    else:
+                        def neg_score(theta):
+                            res = np.mean(self._compute_score(psi_elements, theta))
+                            return res
```

Comment:
@SvenKlaassen Thank's for finding this issue
```suggestion
                        def neg_score(theta):
                            res = - np.mean(self._compute_score(psi_elements, theta))
                            return res
```